test/mocha.opts

```diff
@@ -0,0 +1,5 @@
+--compilers js:babel-register,jsx:babel-register
```

Comment:
this is a good change regardless; i'd be stoked to merge it in a separate PR :-)